### ISSUE TITLE
Fortinet

### ISSUE THREAD

**Issue by AniTexs** [SRC-doo/netbox-ssot#74]
Is it planned to have Fortinet under this project too?

**Comment by bl4ko** [comment]
Yes, the next three external sources that we have locally deployed and are planned are:

- netconf (coreswitch IOS-XE)
- fortinet
- cisco firewall

**Comment by bl4ko** [comment]
Hello, source fortigate has been added in the [latest release](https://github.com/bl4ko/netbox-ssot/releases/tag/v0.5.0).

**Comment by TaLoN1x** [comment]
would be really good If i could configure fortinet integration to create virtual forty firewall appliances instead of devices. This pattern is getting more and more popular especially with cloud-hybrid environments.

**Comment by bl4ko** [comment]
Hello thanks for the suggestion. Could you provide some examples of what do you mean with virtual firewall appliances?

**Comment by TaLoN1x** [comment]
Ideally microservice should be compatible with FortiManager, as large installation rather use for conf management:
https://www.postman.com/fortinet-emea-tac-api-team/workspace/fortinet/collection/22422020-f33ce7c0-31d3-42e0-8ea3-
717d84719c2d

Under Virtual Appliance I thought of ability to deploy Fortinet FW as virtual machine:
https://www.postman.com/fortinet-emea-tac-api-team/workspace/fortinet/collection/22422020-f33ce7c0-31d3-42e0-8ea3-717d84719c2d

**Comment by bl4ko** [comment]
Oh I see, thanks for clarification. The only problem is that I have only single fortigate device deployed locally, so that's why I used fortigate rest api to fetch and sync its data.

To sync objects with FortiManager, I would need some kind of mock API, or hardcoded responses for each endpoint, so I would know how to implement code. Could you maybe provide some mock api values?

**Comment by TaLoN1x** [comment]
I will try :)
